transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUring.java

```diff
@@ -28,6 +28,7 @@ public final class IoUring {
     private static final boolean IORING_SPLICE_SUPPORTED;
     private static final boolean IORING_ACCEPT_NO_WAIT_SUPPORTED;
     private static final boolean IORING_ACCEPT_MULTISHOT_SUPPORTED;
+    private static final boolean IORING_RECV_MULTISHOT_SUPPORTED;
```

Comment:
Regardless being supported: do we want to be able to test with/without it?
OR we would rely on running tests against different OS version which support it/or not?